### PR TITLE
Add linguistics tokens document field writer.

### PR DESCRIPTION
**Pull request by toregge**
@geirst : please review

### CODE REVIEW

File: searchsummary/src/vespa/searchsummary/docsummary/linguistics_tokens_dfw.h [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+#pragma once
+
+#include "docsum_field_writer.h"
+#include <vespa/searchlib/util/token_extractor.h>
+#include <memory>
+
+namespace search::docsummary {
+
+/*
+ * class for writing annotated string field values from document as
```
Comment (geirst) [inline searchsummary/src/vespa/searchsummary/docsummary/linguistics_tokens_dfw.h:12]:
```suggestion
 * Class for writing annotated string field values from document as
```

**Review by geirst — DISMISSED** [review]
👍

File: searchsummary/src/vespa/searchsummary/docsummary/linguistics_tokens_dfw.h (hunk 1 above)
Comment (toregge) [reply searchsummary/src/vespa/searchsummary/docsummary/linguistics_tokens_dfw.h]: Fixed.

**Review by geirst — APPROVED** [review]
👍

**Pull request merged**